src/rars/util/SystemIO.java

```
@@ -102,6 +102,13 @@ private static String readStringInternal(String init, String prompt, int maxleng
         } else {
             if (Globals.getSettings().getBooleanSetting(Settings.Bool.POPUP_SYSCALL_INPUT)) {
                 input = Globals.getGui().getMessagesPane().getInputString(prompt);
+            } else if (!Globals.getGui().getMessagesPane().getInputField().isEmpty()) {
+                try {
+                    input = getInputReaderFromGui().readLine();
+                    if (input == null)
+                        input = "";
+                } catch (IOException e) {
```

Silencing an exception smells fishy here.